### DISCUSSION TITLE
Can't run one more jobs at the same time

### DISCUSSION THREAD

**Discussion by xingzhe1129** [nextpyp/.github#29] [Q&A] [answered]
Hi Everyone,

I am trying to copy a running job and try to use different parameters to run another job in the same project, but it is always pending except killing the first running job, although there is resource available. The second job is not a dependent of the first job but it seems to be that. Does anyone met this? Thanks for your advices and solutions. Best.

**Comment by tofushan** [comment, marked as answer]
Currently we do not enable running more than one jobs **in the same project**. Alternatively, you can run multiple jobs in **different projects**. 

I can see the rationale running two jobs in the same projects to experiment different parameters in parallel rather than in sequence to speed up the process. I add @abartesaghi to this thread for considering this to be a potential feature.

**Reply by xingzhe1129** [reply]
Thank you so much for your quick reply and consider this.

**Comment by asarnow** [comment]
I want to second this and suggest that Runs could be a natural unit for sequential jobs.

**Reply by xingzhe1129** [reply]
I still strongly recommend to add this function.
